app/controllers/api/services_controller.rb

```diff
@@ -69,17 +68,23 @@ def destroy
 
   private
 
-  def build_backend_api_configs
-    return [] if !provider_can_use?(:api_as_product) || params[:service][:backend_api].blank?
-    backend_api = current_account.backend_apis.find(params[:service][:backend_api])
-    [BackendApiConfig.new(backend_api: backend_api)]
-  end
-
   def service_params
     allowed_params = %i[system_name name description]
     params.require(:service).permit(allowed_params)
   end
 
+  private
+
+  def service_params
+    permitted_params = [:name, :system_name, :description, :support_email, :deployment_option, :backend_version,
+      :intentions_required, :buyers_manage_apps, :referrer_filters_required,
+      :buyer_can_select_plan, :buyer_plan_change_permission, :buyers_manage_keys,
```

Comment:
You can automatically fix this bad identation by just doing `rubocop app/controllers/api/services_controller.rb --only Layout/AlignArray -a`